third_party/move/move-binary-format/src/serializer.rs

```diff
@@ -867,22 +867,94 @@ fn serialize_ability_sets(binary: &mut BinaryData, sets: &[AbilitySet]) -> Resul
 fn serialize_function_attributes(
     binary: &mut BinaryData,
     attributes: &[FunctionAttribute],
+    version: u32,
```

Comment:
Convention in this file seems to be `version` as the first param when applicable? Either way, good to be consistent.